pkg/web/api/v1/audit.go

```diff
@@ -119,7 +119,7 @@ type ComplianceAuditLogQuery struct {
 
 	// DetailedLogs will return the full log entry if true (otherwise List
 	// returns only summary info)
-	DetailedLogs bool `json:"detailed_logs,omitempty"`
+	DetailedLogs bool `json:"detailed_logs,omitempty" form:"detailed_logs" url:"detailed_logs,omitempty"`
```

Comment:
Nice fix! It took me forever to realize that not including `form` was the cause of a bug I encountered not long after I first started.